packages/pigz/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "pigz"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/madler/pigz/archive/refs/tags/v2.7.tar.gz"
```

I'd prefer to use the "official" source releases since those have a published hash and GPG signature:
```suggestion
url = "https://zlib.net/pigz/pigz-2.7.tar.gz"
```